cli/types/cypress.d.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference path="./cy-http.d.ts" />
```

Comment:
There is, instead of returning the server I will just return `close`. This was only added to be able to access the webapack-dev-server in the tests (need to ensure the WDS is closed bdfore the test ends).

Updated in https://github.com/cypress-io/cypress/pull/14884